src/components/Home/EditModal.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="post">
+    <fa-icon class="x" @click="close" icon="fa-solid fa-xmark fa-2xl" />
+    <p>
+      <input
+        id="author"
+        class="post-author"
+        placeholder="Nome completo"
+        v-model="updatedSuggestion.author"
+      />
+    </p>
+    <div class="post-content-score-box">
+      <p>
+        <input
+          type="number"
+          min="0"
+          max="10"
+          class="post-score"
+          placeholder="nota"
+          v-model="updatedSuggestion.score"
+        />
+      </p>
+      <div>
+        <textarea class="content" v-model="updatedSuggestion.content" />
+      </div>
+    </div>
+    <button @click="editSuggestion(updatedSuggestion.id, updatedSuggestion)">
+      Editar
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Watch, Emit } from "vue-property-decorator";
+import suggestionService from "@/services/suggestionService";
+@Component({})
+export default class EditModal extends Vue {
```

Comment:
Estamos usando algum componente?
```suggestion
import { Vue, Prop, Watch, Emit } from "vue-property-decorator";
import suggestionService from "@/services/suggestionService";

export default class EditModal extends Vue {
```